model/registry/deprecated/deployment.yaml

```diff
@@ -0,0 +1,22 @@
+groups:
+  - id: registry.deployment.deprecated
+    prefix: deployment
+    type: attribute_group
+    brief: >
+      "Describes deprecated deployment attributes."
+    attributes:
+      - id: environment
+        type: string
+        stability: experimental
+        deprecated: 'Deprecated, use `deployment.environment` instead.'
+        brief: >
+          'Deprecated, use `deployment.environment` instead.'
+        note: |
```

Comment:
you don't need these fields for deprecated attributes. Only updated `brief` and `deprecated`

Reply:
Good to know. Ran into some issues with deprecating so I pretty much copied the pattern from what I saw in `db`

Reply:
The checks require a little bit more than `brief` and `deprecated`.

```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
✖ Invalid attribute definition detected while resolving '"/source/registry/deprecated/deployment.yaml"' (group_id='registry.deployment.deprecated', attribute_id='environment'). This attribute is a string but it does not contain any examples.
make: *** [table-generation] Error 1

WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
✖ The semantic convention spec is invalid (path_or_url: "/source/registry/deprecated/deployment.yaml"). groups[0].attributes: data did not match any variant of untagged enum AttributeSpec at line 8 column 7
make: *** [table-generation] Error 1
```

The basic requirements appear to be:

```yaml
      - id: environment
        type: string
        stability: experimental
        deprecated: 'Deprecated, use `deployment.environment.name` instead.'
        brief: >
          'Deprecated, use `deployment.environment.name` instead.'
        examples: ['staging', 'production']
```